src/components/SettingsTreasuriesBlock.vue

```diff
@@ -16,16 +17,21 @@ const treasuryObj = {
 };
 
 const modalTreasuryOpen = ref(false);
+const modalOsnapOpen = ref(false);
 const currentTreasuryIndex = ref<number | null>(null);
 const currentTreasury = ref<TreasuryWallet>(clone(treasuryObj));
+const hasOsnapPlugin = computed(() => {
```

Comment:
we only show osnap related components and run related logic if the plugin is installed on the space. other users are unaffected